algebird-core/src/main/scala/com/twitter/algebird/CountMinSketch.scala

```diff
@@ -1332,5 +1332,10 @@ object CMSHasherImplicits {
       CMSHasher.hashBytes(a, b, width)(x.getBytes("UTF-8"))
   }
 
+  implicit object CMSHasherBigDecimal extends CMSHasher[BigDecimal] {
+    override def hash(a: Int, b: Int, width: Int)(x: BigDecimal): Int =
+      CMSHasherString.hash(a, b, width)(x.underlying.toPlainString)
```

Comment:
D'oh. Yes of course. Done. 

FWIW, ```x.underlying.unscaledValue.toByteArray``` does allocate a temporary array. From what I understand HotSpot'JIT is doing these days, inline + lifetime escape analysis should eventually make that a (very cheap) stack alloc if that bit of code is hot.
(didn't check the C2 output to confirm)